data-prepper-pipeline-parser/src/resources/templates/documentdb-template.yaml

```diff
@@ -0,0 +1,69 @@
+"{{pipeline-name}}-writer":
```

Comment:
This template is not the final version, needs to be finalized. Check the test/resources/templates/testSource for the templates i used to test

Reply:
acknowledgments: false -> This should be true.